src/contexts/trigger.context.tsx

```diff
@@ -11,11 +11,10 @@ type TriggerProps = { children?: React.ReactNode };
 export const TriggerProvider: React.FC<TriggerProps> = ({ children }) => {
```

Comment:
Why do we need this?

Reply:
this context contains the flag to start the game once user affirms start of game in the modal , and only then timer starts running and turn starts

Reply:
Ok , in that case I think it should be a Game context.We might have others game config their. Like the data that will be gerated from the welcome page for example. Having a provider only for a flag sound a little too much.

Reply:
I agree